### ISSUE TITLE
GitHub workflows break on missing io/fs

### ISSUE THREAD

**Issue by chickenandpork** [masmovil/bazel-rules#47]
Failure in GitHub integration tests:
```
go: finding sigs.k8s.io/yaml v1.3.0
[50](https://github.com/chickenandpork/rules_helm/actions/runs/3473491467/jobs/5805560435#step:4:51)
go: finding gopkg.in/yaml.v2 v2.4.0
[51](https://github.com/chickenandpork/rules_helm/actions/runs/3473491467/jobs/5805560435#step:4:52)
go: finding github.com/google/safetext v0.0.0-20220905092116-b49f7bc46da2
[52](https://github.com/chickenandpork/rules_helm/actions/runs/3473491467/jobs/5805560435#step:4:53)
build sigs.k8s.io/kind: cannot load io/fs: malformed module path "io/fs": missing dot in first path element
[53](https://github.com/chickenandpork/rules_helm/actions/runs/3473491467/jobs/5805560435#step:4:54)
Error: Process completed with exit code 1.
```

`io/fs` is not provided by the PRs being reviewed; instead, the following two lines conflict:
https://github.com/masmovil/bazel-rules/blob/7bd160b5fe0354052e98b1dfb0cc6c4300b58026/.github/workflows/integration-tests.yaml#L17,L18
```
        go-version: 1.13
    - name: Install latest version of Kind
```

- "latest version of kind" uses a recent version of `go >= 1.16` by referencing `io/fs` introduced in go-1.16
- pinned `go == 1.13` doesn't provide it.

**Comment by chickenandpork** [comment]
I have a fix for this (my `20221119-fix-upstream-integration-tests` branch) but testing a fix to a problem this one revealed: missing `go.sum` entry.  One issue, one fix, one PR, I'll file that as a different issue.

**Comment by chickenandpork** [comment]
This was resolved by #44
